svelte-stuff/components/ViewStory.svelte

```diff
@@ -76,13 +77,15 @@
     {/if}
   </div>
   {#if textStory}
-    {#if !textStory.hasLiked}
+    {#if !textStory.hasLiked && likeClickable && currentUserId !== textStory.creatorId}
       <svg
         on:click={async () => {
+          likeClickable = false;
           try {
             await mutation(`/like-text-story/${textStory.id}`, {});
             textStory.hasLiked = true;
             textStory.numLikes++;
+            likeClickable = true;
           } catch {}
         }}
```

Comment:
What if the try doesn't work and it catches, then likeClickable will still be false.

```suggestion
        on:click={async () => {
          likeClickable = false;
          try {
            await mutation(`/like-text-story/${textStory.id}`, {});
            textStory.hasLiked = true;
            textStory.numLikes++;
          } catch {}
          likeClickable = true;
        }}
```